client/src/components/HostPage/index.tsx

```diff
@@ -19,48 +23,117 @@ export function HostPage() {
     // Grab our game code from the global state
     const gameCode = useAppSelector(selectGameCode);
 
-    // Listen for the player join event from gameService and update our state if one joins
+    const [question, setQuestion] = useState(['None']);
+    const [timeRemaining, setTimeRemaining] = useState(0);
+    const [timerActive, setTimerActive] = useState(false);
+
+    // Listen for the player join event from roomService and update our state if one joins
     const handlePlayerJoin = () => {
         if (socketService.socket)
-            gameService.onPlayerJoin(socketService.socket, (playerNames) => {
+            roomService.onPlayerJoin(socketService.socket, (playerNames) => {
                 console.log(playerNames);
-                dispatch({type: 'playerList/set', payload: playerNames}); // Dispatch action to change playerList
-                dispatch({type: 'gameStats/setNumPlayers', payload: playerNames.length}); // Dispatch action to change playerList
+                dispatch({ type: 'playerList/set', payload: playerNames }); // Dispatch action to change playerList
+                dispatch({ type: 'gameStats/setNumPlayers', payload: playerNames.length }); // Dispatch action to change playerList
             });
     };
 
+    // This function is called when the user presses the "host" button
+    const startGame = async (e: React.FormEvent) => {
+
+        // Prevent the page from refreshing
+        e.preventDefault();
+
+        // Get our socket and tell the server to host a room by calling our roomService.hostRoom function
+        const socket: any = socketService.socket;
+        const joined = await gameService.startRound(socket, 1).catch((err) => {
+            alert(err);
+        });
+
+        // Update state variables to display the new host screen
+        if (joined) {
+            console.log(joined);
+            setQuestion(joined);
```

Comment:
Is the connection returning a list of ALL the questions and answers for the game? I thought that the person who lost the round chooses the next category so we wouldn't want a static list but rather pull from a specific category once it's chosen?